dbms/src/Flash/ResourceControl/LocalAdmissionController.h

```diff
@@ -479,6 +485,23 @@ class LocalAdmissionController final : private boost::noncopyable
             if (thread.joinable())
                 thread.join();
         }
+
+        if (need_reset_unique_client_id.load())
+        {
+            try
+            {
+                etcd_client->deleteServerIDFromGAC(unique_client_id);
```

Comment:
It looks like `unique_client_id` has a data race, use `std::atomic_uint64` instead?

Reply:
ok